serializers/src/glTF/2.0/babylon.glTFExporter.ts

```diff
@@ -783,8 +783,12 @@ module BABYLON.GLTF2 {
          * @param babylonTransformNode Babylon mesh used as the source for the transformation data
          */
         private setNodeTransformation(node: INode, babylonTransformNode: TransformNode): void {
-            if (!babylonTransformNode.position.equalsToFloats(0, 0, 0)) {
-                node.translation = this._convertToRightHandedSystem ? _GLTFUtilities._GetRightHandedPositionVector3(babylonTransformNode.position).asArray() : babylonTransformNode.position.asArray();
+            const pivotPoint = babylonTransformNode.getPivotPoint();
+            if (!babylonTransformNode.position.equalsToFloats(0, 0, 0) && !pivotPoint.equalsToFloats(0,0,0)) {

```

Comment:
the pivot point is not exactly a position.
It is more a pre translation (before rotation)
I don't think we can export it as translation

@bghgary is there a notion of Pivot in gltf?

Reply:
Not that I'm aware of.

Reply:
Ok so we need to chat to see how to correctly support pivot point